bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConfiguration.java

```diff
@@ -44,7 +44,7 @@
   }
 
   public static Configuration configure(String projectId, String zoneName, String clusterName) {
-    Configuration config = new Configuration();
```

Comment:
Gotcha.  We needed a programmatic way to configure Dataflow, and this was the way we did it.  It may make sense to discuss additional programmatic idioms for bigtable, but I hope that such discussions are out of scope for this specific change.